docs/docs/eraser-flags.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Customization
+---
+
```

Comment:
let's add this is for eraser specifically. can these be configured through manager?

Reply:
if not, i think we need a way similar to scanner args are passed through

Reply:
they can be passed in like the scanner args, but it applies to both eraser and collector right now. Should we remove the delay for collector pods?

Reply:
addressed eraser/collector pods in #313